main.md

```diff
@@ -191,6 +191,8 @@ The AS MUST process the request as follows:
 2. The AS MUST must reject the request if the `request_uri` authorization request parameter is provided.
 3. The AS MUST validate the request the same way as at the authorization endpoint. For example, the authorization server checks whether the redirect URI matches one of the redirect URIs configured for the `client_id`. It MAY also check whether the client is authorized for the `scope` for which it is requesting access. This validation allows the authorization server to refuse unauthorized or fraudulent requests early. 
 
+The AS MAY allow confidential clients to register per-authorization request redirect URIs. This is possible since, in contrast to [@!RFC6749], PAR gives the AS the ability to authenticate and authorize clients before the actual authorization request is performed. This mechanism is especially useful for clients interacting with a federation of ASs (or OpenID Connect OPs), for example in Open Banking, and are unable to use dynamic client registration to establish AS-specific redirect URIs (as recommended by [@I-D.ietf-oauth-security-topics], section-3.1).  
+
```

Comment:
I wonder if we need to expand a little to explain that the client can authenticate using a federated PKI or similar. Otherwise readers unfamiliar with PSD2 and eIDAS may not be sure how you can have a confidential client that hasn't registered.

Reply:
maybe "use" rather than "register" in that first sentence? Because of dynamic client registration and other language in the OAuth/OIDC world, the word "register" kinda implies data that is persisted with the client record

Reply:
I'm not fully following how registration could or would be skipped in the given situation. But maybe it's just me...

Reply:
It's about the explicit registration with the particular AS.

If the client_id and the cert is managed someplace the AS trusts, the client can send a PAR to the AS without the need to previously explicitly register. The AS will authenticate and authorize based on the data provided in the request. 
eIDAS: the TSP issues a X.509 cert containing the TPP id and the TPP roles. The TPP id is used to build the client id (NextGenPSD2 does it that way). The TPP roles determine what services the TPP is allowed to use.
yes: Clients are registered with us and are issued a client_id. We distribute the client data (including certs) to the ASs. We could have used a PKI instead but we like self-signed certs :-)

The AS will most likely register the client on demand, but that's hidden in the AS implementation.

Reply:
Okay, yeah, I get that. Sorta anyway. I guess what I was trying to say is that I'm having a hard time going from the text in this PR to that. I'm not sure how to improve it. But I do think the points you made in the comment above about avoiding "explicit registration" and being an implementation detail of an AS helped me.

Reply:
better now?

Reply:
explicite -> explicit 

but yeah, that's somewhat better